docs/openapi.yaml

```diff
@@ -93,6 +84,18 @@ paths:
           schema:
             type: string
             example: 'bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U'
+        - in: query
+          name: blockHeight
+          required: false
+          description: Evaluate the contract up to a specific block height. Only applicable if sortKey is not provided.
+          schema:
+            type: number
+        - in: query
+          name: sortKey
+          required: false
+          description: Evaluate the contract at up to a specific sort key. Only applicable if blockHeight is not provided.
+          schema:
+            type: string
```

Comment:
Probably be worth noting what happens If both are provided

Reply:
```
'400':
          description: Bad request if query parameters are invalid.

```          
is defined as a response error